python/finufft/finufft/_finufft.py

```diff
@@ -30,7 +30,8 @@
 lib = None
 # Try to load a local library directly.
 try:
-    lib = ctypes.cdll.LoadLibrary('libfinufft.so')
+    path = os.path.dirname(__file__)
+    lib = ctypes.cdll.LoadLibrary(path+'/libfinufft.so')
```

Comment:
Should probably has `os.path.join` here. Also, how does this work in windows? Are we relying on the compiled C extension to rpath us in the right direction?

Reply:
There might be a cpython API to load a library without specifying path or extension we should dig into it.